compiler/aarch64/codegen/GenerateInstructions.cpp

```diff
@@ -505,6 +505,18 @@ TR::Instruction *generateNegInstruction(TR::CodeGenerator *cg, TR::Node *node,
    return new (cg->trHeapMemory()) TR::ARM64Trg1ZeroSrc1Instruction(op, node, treg, sreg, cg);
    }
 
+TR::Instruction *generateMovBitMaskInstruction(TR::CodeGenerator *cg, TR::Node *node,
+   TR::Register *treg, bool N, uint32_t imm, bool is64bit, TR::Instruction *preced)
+   {
+   /* Alias of ORR instruction */
+   TR::InstOpCode::Mnemonic op = is64bit ? TR::InstOpCode::orrimmx : TR::InstOpCode::orrimmw;
+
+   TR::ARM64Trg1ZeroImmInstruction *intr;
```

Comment:
`instr` is used nowhere in this function.

Reply:
I forgot to remove this line. Thanks!